lib/queries/index.js

```diff
@@ -0,0 +1,3 @@
+const { loadQuery } = require('../utils')
+
+module.exports.REPO_INFO_QUERY = loadQuery('repo_info')
```

Comment:
Better to use

```javascript
export const REPO_INFO_QUERY = // ...
```

or

```javascript
export {
  REPO_INFO_QUERY: // ...
}
```

I think

Reply:
👍 I'll make a PR after this to setup Babel for the backend and start using ES2017 syntax.

Reply:
Alright.